docker/immich_auto_album.sh

```diff
@@ -155,5 +155,9 @@ elif [ "$COMMENTS_AND_LIKES" == "0" ]; then
     args="--comments-and-likes-disabled $args"
 fi
 
+if [ !-z "$UPDATE_ALBUMS_PROP_MODE" ]; then
+    args="--update-albums-prop-mode $UPDATE_ALBUMS_PROP_MODE $args"
```

Comment:
Slight naming mixup for the argument (and the environment variable): In the python script, the argument is called `--update-album-props-mode`, here the argument is called `--update-albums-prop-mode` and the env variable `UPDATE_ALBUMS_PROP_MODE`.
Please update both to match the argument in the python script.